types/stylelint/index.d.ts

```diff
@@ -620,6 +622,8 @@ declare module 'stylelint' {
 		};
 	};
 
+	export type Stylelint = Omit<PublicApi, '_createLinter'>;
+
```

Comment:
[suggest] Let's remove the new type to minimize this pull request change. Perhaps, later refactoring PRs may add similar types.

```suggestion
```